package.json

```diff
@@ -61,6 +61,7 @@
     "karma-rollup-preprocessor": "^7.0.0",
     "karma-safari-private-launcher": "^1.0.0",
     "merge-stream": "^1.0.1",
+    "moment": "^2.10.2",
```

Comment:
should this be a peer dependency?

Reply:
I don't think so. If we make it a peer dependency it will warn users that it will need to be installed. However, it doesn't necessarily need to be installed because you could choose luxon or date-fns instead and so I wouldn't want to give users unnecessary warnings on their console all the time